css/properties/align-items.json

```diff
@@ -6,6 +6,7 @@
           "__compat": {
             "description": "Supported in Flex Layout",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/align-items",
+            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#propdef-align-items",
```

Comment:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#align-items-property",
```

Reply:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#propdef-align-items",
```

Reply:
That section defined two properties, but I guess that's OK.

Reply:
I'll have to merge the suggestions into one in order to apply them:

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#align-items-property",
```

Reply:
We need to decide whether to link to sections or directly to the definitions.

Reply:
```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#align-items-property",
```